### YamlMime:ManagedReference
items:
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  id: FabricBackupDirectoryNotEmptyException
  parent: system.fabric.exception
  children:
  - system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException()
  - system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String)
  - system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String,Throwable)
  - system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String,Throwable,boolean,boolean)
  - system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(Throwable)
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  langs:
  - java
  name: FabricBackupDirectoryNotEmptyException
  nameWithType: FabricBackupDirectoryNotEmptyException
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  type: Class
  source:
    remote: &o0
      path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
      branch: master
      repo: https://msazure.visualstudio.com/One/_git/sf-java
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
    startLine: 5
  package: system.fabric.exception
  summary: "<p>The exception that is thrown when the user provided backup directory is not empty. </p>"
  syntax: &o1
    content: public class FabricBackupDirectoryNotEmptyException extends FabricException
  inheritance:
  - java.lang.Object
  - RuntimeException
  - system.fabric.exception.FabricException
  inheritedMembers:
  - system.fabric.exception.FabricException.FabricException()
  - system.fabric.exception.FabricException.FabricException(FabricErrorCode)
  - system.fabric.exception.FabricException.FabricException(String)
  - system.fabric.exception.FabricException.FabricException(String,FabricErrorCode)
  - system.fabric.exception.FabricException.FabricException(String,Throwable)
  - system.fabric.exception.FabricException.FabricException(String,Throwable,FabricErrorCode)
  - system.fabric.exception.FabricException.FabricException(Throwable)
  - system.fabric.exception.FabricException.FabricException(Throwable,FabricErrorCode)
  - system.fabric.exception.FabricException.FabricException(String,Throwable,boolean,boolean)
  - system.fabric.exception.FabricException.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  - system.fabric.exception.FabricException.getErrorCode()
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException()
  id: FabricBackupDirectoryNotEmptyException()
  parent: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  langs:
  - java
  name: FabricBackupDirectoryNotEmptyException()
  nameWithType: FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException()
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException()
  overload: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
    startLine: 13
  package: system.fabric.exception
  summary: "<p>Initializes a new instance of the <xref uid=\"system.fabric.exception.FabricBackupDirectoryNotEmptyException\" data-throw-if-not-resolved=\"false\" data-raw-source=\"FabricBackupDirectoryNotEmptyException\"></xref> class. </p>"
  syntax:
    content: public FabricBackupDirectoryNotEmptyException()
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String)
  id: FabricBackupDirectoryNotEmptyException(String)
  parent: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  langs:
  - java
  name: FabricBackupDirectoryNotEmptyException(String message)
  nameWithType: FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String message)
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String message)
  overload: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
    startLine: 39
  package: system.fabric.exception
  summary: "<p>Initializes a new instance of the <xref uid=\"system.fabric.exception.FabricBackupDirectoryNotEmptyException\" data-throw-if-not-resolved=\"false\" data-raw-source=\"FabricBackupDirectoryNotEmptyException\"></xref> class with a specified error message.</p>\r\n<p></p>"
  syntax:
    content: public FabricBackupDirectoryNotEmptyException(String message)
    parameters:
    - id: message
      type: "26831127"
      description: <p>input message </p>
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String,Throwable)
  id: FabricBackupDirectoryNotEmptyException(String,Throwable)
  parent: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  langs:
  - java
  name: FabricBackupDirectoryNotEmptyException(String message, Throwable cause)
  nameWithType: FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String message, Throwable cause)
  overload: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
    startLine: 28
  package: system.fabric.exception
  summary: "<p>Initializes a new instance of the <xref uid=\"system.fabric.exception.FabricBackupDirectoryNotEmptyException\" data-throw-if-not-resolved=\"false\" data-raw-source=\"FabricBackupDirectoryNotEmptyException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.</p>\r\n<p></p>"
  syntax:
    content: public FabricBackupDirectoryNotEmptyException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
      description: <p>The error message that explains the reason for the exception. </p>
    - id: cause
      type: a04fafd7
      description: <p>The exception that is the cause of the current exception, or a null reference if no inner exception is specified. </p>
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String,Throwable,boolean,boolean)
  id: FabricBackupDirectoryNotEmptyException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  langs:
  - java
  name: FabricBackupDirectoryNotEmptyException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  nameWithType: FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  overload: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
    startLine: 47
  package: system.fabric.exception
  syntax:
    content: protected FabricBackupDirectoryNotEmptyException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(Throwable)
  id: FabricBackupDirectoryNotEmptyException(Throwable)
  parent: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  langs:
  - java
  name: FabricBackupDirectoryNotEmptyException(Throwable cause)
  nameWithType: FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(Throwable cause)
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException(Throwable cause)
  overload: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException*
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricBackupDirectoryNotEmptyException.java
    startLine: 43
  package: system.fabric.exception
  syntax:
    content: public FabricBackupDirectoryNotEmptyException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
references:
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException*
  name: FabricBackupDirectoryNotEmptyException
  nameWithType: FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException.FabricBackupDirectoryNotEmptyException
  package: system.fabric.exception
- uid: "26831127"
  spec.java:
  - name: String
    fullName: String
- uid: a04fafd7
  spec.java:
  - name: Throwable
    fullName: Throwable
- uid: 4fc6e284
  spec.java:
  - name: boolean
    fullName: boolean
- uid: system.fabric.exception.FabricException.FabricException()
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException()
  nameWithType: FabricException.FabricException()
  fullName: system.fabric.exception.FabricException.FabricException()
  type: Constructor
  syntax:
    content: public FabricException()
- uid: system.fabric.exception.FabricException.FabricException(FabricErrorCode)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(FabricErrorCode errorCode)
    parameters:
    - id: errorCode
      type: system.fabric.FabricErrorCode
- uid: system.fabric.exception.FabricException.FabricException(String)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(String message)
  nameWithType: FabricException.FabricException(String message)
  fullName: system.fabric.exception.FabricException.FabricException(String message)
  type: Constructor
  syntax:
    content: public FabricException(String message)
    parameters:
    - id: message
      type: "26831127"
- uid: system.fabric.exception.FabricException.FabricException(String,FabricErrorCode)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(String message, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: errorCode
      type: system.fabric.FabricErrorCode
- uid: system.fabric.exception.FabricException.FabricException(String,Throwable)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(String message, Throwable cause)
  nameWithType: FabricException.FabricException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception.FabricException.FabricException(String,Throwable,FabricErrorCode)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric.FabricErrorCode
- uid: system.fabric.exception.FabricException.FabricException(Throwable)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(Throwable cause)
  nameWithType: FabricException.FabricException(Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception.FabricException.FabricException(Throwable,FabricErrorCode)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(Throwable cause, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric.FabricErrorCode
- uid: system.fabric.exception.FabricException.FabricException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  nameWithType: FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception.FabricException.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  nameWithType: FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
    - id: errorCode
      type: system.fabric.FabricErrorCode
- uid: system.fabric.exception.FabricException.getErrorCode()
  parent: system.fabric.exception.FabricException
  href: system.fabric.exception.FabricException.yml
  name: getErrorCode()
  nameWithType: FabricException.getErrorCode()
  fullName: system.fabric.exception.FabricException.getErrorCode()
  type: Method
  syntax:
    content: public FabricErrorCode getErrorCode()
    return:
      type: system.fabric.FabricErrorCode
- uid: system.fabric.exception.FabricException
  parent: system.fabric.exception
  href: system.fabric.exception.FabricException.yml
  name: FabricException
  nameWithType: FabricException
  fullName: system.fabric.exception.FabricException
  type: Class
  syntax:
    content: public class FabricException
- uid: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  parent: system.fabric.exception
  href: system.fabric.exception.FabricBackupDirectoryNotEmptyException.yml
  name: FabricBackupDirectoryNotEmptyException
  nameWithType: FabricBackupDirectoryNotEmptyException
  fullName: system.fabric.exception.FabricBackupDirectoryNotEmptyException
  type: Class
  summary: "<p>The exception that is thrown when the user provided backup directory is not empty. </p>"
  syntax: *o1
